volumes/vds/sql/delete/delete-counts_15min.sql

```diff
@@ -0,0 +1,4 @@
+DELETE FROM vds.counts_15min
+WHERE
+    datetime_15min >= '{{ ds }} 00:00:00'::timestamp
```

Comment:
You can hide these errors (as long as we know they are actually not errors) by adding `-- noqa: TMP` at the end of the line OR `-- noqa: disable=TMP` at the top of the file